charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.1.0
+version: 2.1.1
```

Please bump the minor version here as we are adding a new feature.
```suggestion
version: 2.2.0
```